src/tiled/commandmanager.cpp

```diff
@@ -40,6 +40,7 @@ CommandManager::CommandManager()
     , mLogger(new LoggingInterface())
 {
     updateActions();
+    retranslateUi();
```

Comment:
Erm, while I did say this line should be in the constructor, when I said that I didn't realize that the action was actually not created here, but in `CommandManager::updateActions`. As such, the call to `retranslateUi` should of course be in `updateActions`, otherwise the text isn't set when `updateActions` is called in response to the commands changing.

Reply:
:man_facepalming: